### ISSUE TITLE
[module:install] Problem with core module without dependecies - language module

### ISSUE THREAD

**Issue by vincenzodibiaggio** [hechoendrupal/DrupalConsole#1578]
```
# bin/drupal module:install language  
Recoverable fatal error: Argument 1 passed to Drupal\Core\ProxyClass\Extension\ModuleInstaller::install() must be of the type array, null given, called in [...]/vendor/drupal/console/src/Command/Module/InstallCommand.php on line 133 and defined in [...]/web/core/lib/Drupal/Core/ProxyClass/Extension/ModuleInstaller.php on line 85
Recoverable fatal error: Argument 1 passed to Drupal\Core\ProxyClass\Extension\ModuleInstaller::install() must be of the type array, null given, called in [...]/vendor/drupal/console/src/Command/Module/InstallCommand.php on line 133 and defined in Drupal\Core\ProxyClass\Extension\ModuleInstaller->install() (line 85 of [...]/web/core/lib/Drupal/Core/ProxyClass/Extension/ModuleInstaller.php).
```

**Comment by jmolivas** [comment]
@vincenzodibiaggio: Thanks for reporting this, but ame as issue #1580, this was fixed with PR #1592.

**Comment by vincenzodibiaggio** [comment]
:+1:
